.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
I was thinking about that but since this is the default configuration and all config that is committed is actually related to this project, I would prefer to leave these files in.
If someone is using vscode, it would probably be nice to set those defaults as well

Reply:
There could be conflicts in these files if people use different IDE versions or change their configuration. Of course, it's always possible to exclude local modifications to these files from commits.
I don't know how often these files actually change. We could leave them in the repository for now, but if they become too annoying, I'd suggest we remove them.